routing/redirect_in_config.rst

```diff
@@ -96,7 +96,7 @@ Redirecting Using a Route
 
 Assume you are migrating your website from WordPress to Symfony, you want to
 redirect ``/wp-admin`` to the route ``sonata_admin_dashboard``. You don't know
-the path, only the route name. This can be achieved using the
+the path, only the route name, if you pass query parameters to this route and active the ``keepQueryParams``, it will be redirected too. This can be achieved using the
```

Comment:
We usually split lines after the first word that crosses the 72nd character. But we can make that tweak while merging. :)